tex/alg-NT/dedekind.tex

```diff
@@ -169,6 +169,12 @@ \section{Dedekind domains}
 	\emph{every nonzero prime ideal of $\mathcal A$ is in fact maximal}.
 	(The last condition is the important one.)
 \end{definition}
+
+\begin{remark}
+	Note that $\mathcal A$ is a Dedekind domain if and only if $\mathcal A = \OO_K$ for some field
+	$K$, as we will prove below. We're just defining this term for historical reasons\dots
```

Comment:
Which feels slightly silly, but I don't see much harm either.
Without defining Dedekind domain, the theorems would read

* 𝒪_K is Noetherian, integrally closed, and "dim Spec 𝒪_K = 1" i.e. prime ideals are maximal.
* ⟹ prime ideal factorization works in 𝒪_K.

Reply:
Seems fine.